lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -146,8 +151,19 @@ async function runSmokeTest(smokeTestDefn, testOptions) {
 
     // Run Lighthouse.
     try {
+      /** @type {NodeJS.Timeout} */
+      let timeout;
```

Comment:
I mentioned this out of band, but for tracking here:

This adds some timeout redundancy here, since Lighthouse itself will also fail with an error after it times out, which leaves it somewhat unclear where modified timeouts should be specified, here or in the runners. It's not a huge deal, (100s is pretty long, so the core Lighthouse one will always end first), but simplicity-wise it might make sense to move this timeout into the new `devtools.js` and leave runs with the other runners unchanged.

This was mostly a nit before, but the introduction of `beforeAll` makes it a bigger win simplicity-wise, since `devtools.js` wouldn't need a `beforeAll` at that point, it could just call those lines itself and just exempt them from the overall timeout.